app/policies/kudo_policy.rb

```diff
@@ -0,0 +1,9 @@
+class KudoPolicy < ApplicationPolicy
+  def update?
+    true if Time.current <= record.created_at.advance(minutes: 5)
+  end
+
+  def destroy?
+    true if Time.current <= record.created_at.advance(minutes: 5)
```

Comment:
why not simply:

```suggestion
    update?
```

Reply:
This is a smart approach. I changed it. Thanks for CR! ;) 👍